README.md

```diff
@@ -1 +1,111 @@
-### Webex SDK Component Adapter
+<h1 align='center' style='border-bottom: none;'>Webex SDK Component Adapter</h1>
+<h3 align='center'>Webex adapter to share SDK data with the <a href="https://github.com/webex/components">components</a></h3>
+<p align='center'>
+<a href='https://circleci.com/gh/webex/sdk-component-adapter'>
+    <img alt='CircleCI' src='https://circleci.com/gh/webex/sdk-component-adapter.svg?style=shield'>
+  </a>
+  <a href='https://www.npmjs.com/package/@webex/sdk-component-adapter'>
+    <img alt='npm latest version' src='https://img.shields.io/npm/v/@webex/sdk-component-adapter?label=npm%40latest'>
+  </a>
+  <a href='#badge'>
+    <img alt='semantic-release' src='https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg'>
+  </a>
+  <a href='https://github.com/webex/sdk-component-adapter/blob/master/LICENSE'>
+    <img src='https://img.shields.io/npm/l/webex.svg' alt='license'>
+  </a>
+</p>
+
+**Webex SDK Adapter** is a library of adapters to provide live data from [Webex JavaScript SDK](https://github.com/webex/webex-js-sdk) source to [Webex Components](https://github.com/webex/components).
```

Comment:
"live data from sdk source" sounds off to me. How about "live data from the sdk to webex"